src/offset.js

```diff
@@ -103,7 +103,7 @@ jQuery.fn.extend( {
 		rect = elem.getBoundingClientRect();
```

Comment:
For width and height measurements [we use getClientRects](https://github.com/jquery/jquery/blob/ff1a0822f72d2b39fac691dfcceab6ede5623b90/src/css.js#L124-129) to determine if the element is disconnected, before calling getBoundingClientRect, it seems like this IE11 problem might apply here too?